maestro-app/src/main/resources/config/application.yml

```diff
@@ -49,7 +49,7 @@ maestro:
       organization: ICGC
       country: CA
     # you can other SONGs as needed
-    - code: song.overture
+    - code: song.overture1
```

Comment:
Can you explain why code is changed to `song.overture1`?

Reply:
typo will fix it

Reply:
actually it is not a typo, i changed the first entry as Dusan mentioned in his comment so it seems correct to keep.